modules/core/test/test_mat.cpp

```diff
@@ -2432,6 +2439,7 @@ TEST(Mat1D, DISABLED_basic)
         Mat m(5, 100, CV_8UC1, Scalar::all(0));
         const Mat row2D = m.row(2);
         EXPECT_NO_THROW(m1.copyTo(row2D));
+        EXPECT_NO_THROW(row2D.copyTo(m1_copy));
```

Comment:
Idea is to write set of simple independent tests with common predefined variables.
This line violates that rule as it modifies common variable (result affects checks below).